.github/dependabot.yml

```diff
@@ -55,6 +55,35 @@ updates:
         patterns:
           - "k8s.*"
           - "sigs.k8s*"
+      misc-dependencies:
+        patterns:
+          - "*"
+  # API Go module
+  - package-ecosystem: "gomod"
+    directory: "/api/"
+    schedule:
+      interval: "weekly"
+      day: "friday"
+      time: "01:00"
+      timezone: "Etc/UTC"
+    commit-message:
+      prefix: "NO-JIRA"
+    labels:
+      - "area/api"
```

Comment:
This isn't a real label for us. Maybe use `area/ci-tooling`?

Reply:
makes sense